docs/UserGuide.md

```diff
@@ -69,31 +69,210 @@ The following annotated text-boxes are used throught this guide to provide usefu
+
+- **Command Box:** Where you can type and enter commands.
+- **Result Display:** Shows the result of the command you entered.
+- **Patient List:** Displays all the patients currently stored in the system.
+- **Appointment List:** Displays the scheduled appointments for patients.
+- **Menu (File/Help):** Provides additional options for managing the app (e.g., exit, access help).
+- **Data Storage Location Footer:** Displays the location where patient and appointment data are stored.
+
+![GUIOverview.jpg](images%2FGUIOverview.jpg)
+
+---
+
+## Overview of CLI
+
+MediBase3 can be controlled using commands in the **Command Line Interface (CLI)**.
+
+### CLI Command Structure
+
+Each command follows this structure:
+
+![CommandPromptFormat.png](images%2FCommandPromptFormat.png)
+
+Here’s a breakdown of the parts:
+
+| **Component** | **Description**                                    |
+|---------------|----------------------------------------------------|
+| **Command**   | The action you want to perform (e.g., `add`, `edit`) |
+| **Prefix**    | Identifies the data being entered (e.g., `n/` for name) |
+| **Parameter** | The value associated with the prefix (e.g., `John Doe`) |
+
+### Example:
+
+add n/John Doe p/98765432 e/johndoe@example.com
+- **Command:** `add`
+- **Prefix:** `n/` (name), `p/` (phone), `e/` (email)
+- **Parameter:** `John Doe`, `98765432`, `johndoe@example.com`
+
+---
+
+## A First Look at MediBase3
+
+Here’s a basic tutorial on how to start using the application:
+
+1. **View the Help Guide:**
+   - Once MediBase3 is running, you can type the following command to view the help window:
+     ```
+     help
+     ```
+2. **Add a New Patient:**
+   - To add a new patient, use the following command:
+     ```
+     add n/John Doe i/S1234567A d/2000-01-01 g/M p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 
+     ```
+   - This will add a patient named John Doe to the system.
+
+
+3. **Edit Patient Information:**
+   - If there’s a typo or information changes, use the `edit` command:
+     ```
+     edit S1234567A p/91234567 e/johndoe@example.com
+     ```
+   - This will change the phone number and email of the patient with NRIC S1234567A
+
+
+4. **Find a Patient by NRIC:**
+   - To find a patient by NRIC, type:
+     ```
+     findNric T0123456A
+     ```
+   - This will show the patient with the NRIC `T0123456A`.
+
+
+5. **Delete a Patient:**
+   - To remove a patient who is no longer visiting the clinic/hospital, type:
+     ```
+     delete T0123456A
+     ```
+   - This deletes the patient with the NRIC `T0123456A`.
+
+
+6. **Clear All Entries:**
+   - To remove all patient and appointment data from the system, type:
+     ```
+     clear
+     ```
+
+---
+
+Now you’re ready to use MediBase3!
+For a comprehensive list of features and how to use them, [click here!](#features).
+If you have eny more common questions, [click here!](#faq).
```

Comment:
I think this is fine as a quick tutorial for now, maybe in the next milestone we can add some pictures to show and indicate the expected UI feedback

Reply:
Yep can add once we finalise the UI